Python/PRP/Message/pyTimerCallbackMsg.cpp

```diff
@@ -0,0 +1,150 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <PyPlasma.h>
+#include <PRP/Message/plTimerCallbackMsg.h>
+#include "pyTimerCallbackMsg.h"
+#include "pyMessage.h"
+
+extern "C" {
+
+static PyObject* pyTimerCallbackMsg_new(PyTypeObject* type, PyObject* args, PyObject* kwds) {
+    pyTimerCallbackMsg* self = (pyTimerCallbackMsg*)type->tp_alloc(type, 0);
+    if (self != NULL) {
+        self->fThis = new plTimerCallbackMsg();
+        self->fPyOwned = true;
+    }
+    return (PyObject*)self;
+}
+
+static PyObject* pyTimerCallbackMsg_getID(pyTimerCallbackMsg* self, void*) {
+    return PyInt_FromLong(self->fThis->getID());
+}
+
+static PyObject* pyTimerCallbackMsg_getTime(pyTimerCallbackMsg* self, void*) {
+    return PyFloat_FromDouble(self->fThis->getTime());
+}
+
+static int pyTimerCallbackMsg_setID(pyTimerCallbackMsg* self, PyObject* value, void*) {
+    if (value == NULL || !PyInt_Check(value)) {
+        PyErr_SetString(PyExc_TypeError, "ID expects an int");
+        return -1;
+    }
+    self->fThis->setID(PyInt_AsLong(value));
+    return 0;
+}
+
+static int pyTimerCallbackMsg_setTime(pyTimerCallbackMsg* self, PyObject* value, void*) {
+    if (value == NULL || !PyFloat_Check(value)) {
+        PyErr_SetString(PyExc_TypeError, "time expects a float");
+        return -1;
+    }
+    self->fThis->setTime((float)PyFloat_AsDouble(value));
+    return 0;
+}
+
+static PyGetSetDef pyTimerCallbackMsg_GetSet[] = {
+    { _pycs("ID"), (getter)pyTimerCallbackMsg_getID, (setter)pyTimerCallbackMsg_setID, NULL, NULL },
```

Comment:
I was trying to be consistent with other parts of the code I knew about, such as pyVector3's upper case X, Y, and Z when the C++ variable was upper as well. I couldn't find any instances of ID or id in PyHSPlasma to offer guidance here, unfortunately.